arkane/statmech.py

```diff
@@ -1551,3 +1551,15 @@ def getEigvals(self):
        """
        out = subprocess.check_call(['python2',os.environ['Q2DTor'],self.name,'--tor2dns'],
                        cwd=self.q2dtor_path)
+       self.readEigvals()
+
+   def readEigvals(self):
+       """
+       reads an available .evals file to get the QM energy levels
+       for the 2D-NS rotors
+       """
+       f = open(os.path.join(self.q2dtor_path,'IOfiles',self.name+'.evals'),'rU')
```

Comment:
I think `open(... 'U')` is deprecated, [take a look](https://docs.python.org/3.5/library/functions.html#open)

Reply:
fixed